aw/Extensions/Reach.swift

```diff
@@ -0,0 +1,102 @@
+import Foundation
+import MapKit
+
+extension Reach {
+
+    var readingFormatted: String {
+        guard let lastGageReading = lastGageReading, let unit = unit else {
+            //print(self.lastGageReading, self.unit)
+            return "n/a"
+        }
+        return "\(lastGageReading) \(unit)"
+    }
+
+    var color: UIColor {
+        let con = Condition.fromApi(condition: condition ?? "")
```

Comment:
What is condition? Would be better to have a more specific name.

Reply:
`Condition` is what is returned from the api for the level, aka 'low', 'med', 'high' or 'below runnable', 'runnable', 'above runnable'. I tried to match the api naming, even when it wasn't the clearest. `Level` could work, but might make more sense for the more detailed `rc` values.

Reply:
Ah right, I used `FlowLevel` on Android.